### PR TITLE
NXBT-3243: Docs disable github multibranch status

### PR DESCRIPTION
**Pull request by doulba**

### CODE REVIEW

File: docs/platform-setup.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Setup of the platform Jenkins X Team
+
+## Prerequisite
+
+- Jenkins is installed.
+- Plugin disable-github-multibranch-status is installed.
```
Comment (ataillefer) [inline docs/platform-setup.md]:
```suggestion
- The [Disable GitHub Multibranch Status](https://plugins.jenkins.io/disable-github-multibranch-status/) plugin is installed.
```

File: docs/platform-setup.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Disable github multibranch status
```
Comment (ataillefer) [inline docs/platform-setup.md]:
```suggestion
## Disable GitHub Multibranch Status
```
Titles should always be capitalized.

File: docs/platform-setup.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Open [Jenkins](https://jenkins.platform.dev.nuxeo.com/) for platform team.
```
Comment (ataillefer) [inline docs/platform-setup.md]:
```suggestion
Open the platform team's [Jenkins](https://jenkins.platform.dev.nuxeo.com/).
```

File: docs/platform-setup.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+for each jenkins job you want to disable github multibranch status:
```
Comment (ataillefer) [inline docs/platform-setup.md]:
```suggestion
For each, disable the GitHub multibranch status:
```

File: docs/platform-setup.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- open multibranch pipeline job [nuxeo](https://jenkins.platform.dev.nuxeo.com/job/nuxeo/job/nuxeo/configure)
```
Comment (ataillefer) [inline docs/platform-setup.md]:
```suggestion
- open the multibranch pipeline configuration, for instance: [nuxeo](https://jenkins.platform.dev.nuxeo.com/job/nuxeo/job/nuxeo/configure)
```

File: docs/platform-setup.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Click on "Add Behavior" Button
```
Comment (ataillefer) [inline docs/platform-setup.md]:
```suggestion
- In Branch Sources > GitHub > Behaviours, click on the "Add" button.
```

File: docs/platform-setup.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- Select item "Disable Github Notifications"
```
Comment (ataillefer) [inline docs/platform-setup.md]:
```suggestion
- Select "Disable Github Notifications".
```

File: docs/platform-setup.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+- Click on "Save" Button
```
Comment (ataillefer) [inline docs/platform-setup.md]:
```suggestion
- Click on "Save".
```

File: docs/platform-setup.md (hunk 5 above)
Comment (kevinleturc) [reply docs/platform-setup.md]: Text is not exactly the suggestion.

Comment (kevinleturc) [reply docs/platform-setup.md]: You pushed just after my comment.

**Pull request merged**